tests/lib/rules/no-control-regex.js

```diff
@@ -28,6 +28,7 @@ ruleTester.run("no-control-regex", rule, {
         "new (function foo(){})('\\x1f')"
     ],
     invalid: [
+        { code: "var regex = /\\x1f/", errors: [{ message: "Unexpected control character in regular expression.", type: "Literal"}] },
         { code: "var regex = /\\\x1f/", errors: [{ message: "Unexpected control character in regular expression.", type: "Literal"}] },
```

Comment:
Actually I am not sure about this. `/\\x1f/` is the same from the [rules documentation](http://eslint.org/docs/rules/no-control-regex). Or am I missing something ?